app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PPOCardView.kt

```diff
@@ -0,0 +1,616 @@
+                    sendAMessageClickAction = {},
+                    showBadge = true,
+                    onActionButtonClicked = {},
+                    timeNumberForAction = 7
+                )
+
+                Spacer(modifier = Modifier.height(dimensions.paddingMedium))
+            }
+
+            item {
+                PPOCardView(
+                    viewType = PPOCardViewType.CARD_AUTHENTICATED,
+                    onCardClick = {},
+                    projectName = "Sugardew Island - Your cozy farm shop let’s pretend this is a longer title let’s pretend this is a longer title",
+                    pledgeAmount = "$60.00",
+                    creatorName = "Some really really really really really really really long name",
+                    sendAMessageClickAction = {},
+                    showBadge = false,
+                    onActionButtonClicked = {},
+                    timeNumberForAction = 7
+                )
+
+                Spacer(modifier = Modifier.height(dimensions.paddingMedium))
+            }
+
+            item {
+                PPOCardView(
+                    viewType = PPOCardViewType.TAKE_SURVEY,
+                    onCardClick = {},
+                    projectName = "Sugardew Island - Your cozy farm shop let’s pretend this is a longer title let’s pretend this is a longer title",
+                    pledgeAmount = "$70.00",
+                    creatorName = "Some really really really really really really really long name",
+                    sendAMessageClickAction = {},
+                    showBadge = true,
+                    onActionButtonClicked = {},
+                    timeNumberForAction = 8
+                )
+
+                Spacer(modifier = Modifier.height(dimensions.paddingMedium))
+            }
+
+            item {
+                PPOCardView(
+                    viewType = PPOCardViewType.SURVEY_SUBMITTED,
+                    onCardClick = {},
+                    projectName = "Sugardew Island - Your cozy farm shop let’s pretend this is a longer title let’s pretend this is a longer title",
+                    pledgeAmount = "$70.00",
+                    creatorName = "Some really really really really really really really long name",
+                    sendAMessageClickAction = {},
+                    showBadge = false,
+                    onActionButtonClicked = {},
+                    timeNumberForAction = 8
+                )
+
+                Spacer(modifier = Modifier.height(dimensions.paddingMedium))
+            }
+        }
+    }
+}
+
+enum class PPOCardViewType {
+    CONFIRM_ADDRESS,
+    ADDRESS_CONFIRMED,
+    FIX_PAYMENT,
+    PAYMENT_FIXED,
+    AUTHENTICATE_CARD,
+    CARD_AUTHENTICATED,
+    TAKE_SURVEY,
+    SURVEY_SUBMITTED
+}
+
+@Composable
+fun PPOCardView(
+    viewType: PPOCardViewType,
+    onCardClick: () -> Unit,
+    projectName: String? = null,
```

Comment:
@mtgriego how about sending the project itself or a subclass of it, instead of all the individual paramenters?. Probably a subclass that is populated with all the params that are not call to actions might make it simpler